### DISCUSSION TITLE
2 Gits return different status

### DISCUSSION THREAD

**Discussion by weeebdev** [microsoft/WSL#6111] [Q&A] [answered]
When I type 
```bash
git status
```
in Windows filesystem in shows me that there are many changes, but when I type
```bash
git.exe status
```
it shows me that everything okay and there is no any change. Why? I just wanted to switch git depending on the filesystem like [here](https://gist.github.com/jasonboukheir/3fdab92ece236744528447a4f7f5de00).

**Comment by craigloewen-msft** [comment]
That seems strange. I just tried to reproduce this on a project and wasn't able to. Do you have any steps that I could take to try and repro this?

**Comment by Biswa96** [comment, marked as answer]
I think this is same with crlf/lf conflicts. There was an issue/discussion with this type of situation. Try to play with the `core.autocrlf` option in `.gitconfig`.

**Reply by weeebdev** [reply]
I guess that. I turned on the autocrlf and it worked. I mean, there is no mythical changes in `git status` anymore, but now vscode from ubuntu sees them. And I guess now git will automatically convert crlf to lf and spam in the terminal? Can I change the default to LF on windows vscode?
